code/modules/jobs/job_types/other/merc_classes/bogwalker.dm

```diff
@@ -0,0 +1,58 @@
+/datum/advclass/mercenary/bogwalker
+	name = "Bogwalker"
+	tutorial = "You've spent your years wandering the bogs of Psydonia, eking out a living a hunter of both men and beast. Your axe has claimed many a head and the bog has hardened your body and mind against all threats."
```

Comment:
iirc you can split up strings/text with \
useful when text gets long, like tutorials. Helps with making the code easier to read.

Reply:
iirc; \n is for line breaks to show up in the text, and \ is more like a non-existent line break that's done for code clarity
for example, royal knight
<img width="1155" height="171" alt="image" src="https://github.com/user-attachments/assets/3d1f8481-73e0-4ab8-b613-a7fc20ee386c" />
<img width="491" height="205" alt="image" src="https://github.com/user-attachments/assets/a5427677-569d-49e8-8aff-46803749326f" />